ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -97,7 +97,7 @@ Item {
         color: d.isQuote ? Theme.palette.baseColor1 : Theme.palette.directColor1
         font.family: Theme.palette.baseFont.name
         font.pixelSize: Theme.primaryTextFontSize
-        textFormat: root.stripHtmlTags ? Text.PlainText : Text.RichText
+        textFormat: Text.RichText
```

Comment:
what's the need for the `stripHtmlTags` flag in general and why it was used here? Maybe now it could be removed as well?

Reply:
Good question... Gonna find out :)

Reply:
Ok, so it's set to `true` while displaying a reply (inside StatusMessageReply.qml) and when it is, we do here in this component:
```qml
if (root.stripHtmlTags)
  formattedMessage = Utils.stripHtmlTags(formattedMessage)
...
if (root.stripHtmlTags)
  // short return not to add styling when no html
  return formattedMessage
```
basically to strip any HTML formatting (like the bold text in the screenshot). 

The problem here was that emojis are still HTML encoded

Reply:
Meaning, we want to strip the HTML (user) formatting but we still have to use rich text since some stuff is still HTML encoded behind the scenes by the TextEdit